.travis.yml

```diff
@@ -1,39 +1,43 @@
 matrix:
   include:
-    - language: java
-      sudo: false
-      before_install:
-        - "export DISPLAY=:99.0"
-        - "sh -e /etc/init.d/xvfb start"
-      script:
-        - 'mvn clean verify'
-        - 'if [[ $TRAVIS_PULL_REQUEST == "false" && $TRAVIS_BRANCH == "master" ]]; then mvn sonar:sonar -Dsonar.login=${SONAR_TOKEN} -Dsonar.projectKey="camel-lsp-server" -Dsonar.projectName="Camel LSP Server"; fi'
-      jdk: oraclejdk8
-      deploy:
-        provider: releases
-        api_key:  ${DEPLOY_TOKEN}
-        file:
-          - "target/camel-lsp-server-1.0.0-SNAPSHOT.jar"
-        skip_cleanup: true
-        overwrite: true
-        on:
-          branch:  master
-          # tags: true
-        branches:
-          only:
-            - master
-      cache:
-        directories:
-          - $HOME/.m2
-          - $HOME/.sonar/cache
-          - $HOME/.gradle/
-#      after_success:
-#	  - ./cd/before-deploy.sh
-#	  - ./cd/deploy.sh
-env: DISPLAY=:99
+  - language: java
+    sudo: false
+    before_install:
+    - export DISPLAY=:99.0
+    - sh -e /etc/init.d/xvfb start
+    script:
+    - mvn clean verify
+    - if [[ $TRAVIS_PULL_REQUEST == "false" && $TRAVIS_BRANCH == "master" ]]; then
+      mvn sonar:sonar -Dsonar.login=${SONAR_TOKEN} -Dsonar.projectKey="camel-lsp-server"
+      -Dsonar.projectName="Camel LSP Server"; fi
+    jdk: oraclejdk8
+    deploy:
+      provider: releases
+      api_key: "${DEPLOY_TOKEN}"
+      file:
+      - target/camel-lsp-server-1.0.0-SNAPSHOT.jar
+      skip_cleanup: true
+      overwrite: true
+      on:
+        branch: master
+      branches:
+        only:
+        - master
+    cache:
+      directories:
+      - "$HOME/.m2"
+      - "$HOME/.sonar/cache"
+      - "$HOME/.gradle/"
+env:
+  matrix:
+  - DISPLAY=:99
+  global:
+  - secure: F9vg7J7rrSzgNM31AkhpcazyDTR8R7f36WXauJiXJpTA3AXk0/Iq9ntYoD8OZX1eYKWKkEL/k03Iq9DuxEj0Tk7kRXlAv1k7gxLN9cSW0Q7wRO1TcGo0gPt4a1IdDwTpvTGfu7wniRt9qmxVhr0Vox3QBVnIE2KNHVcFiFioGTczlwKdr3oa3NPXdij6XcucHutTmThy3VOpeNhScqUyEFGaimKORGgCdCBouALp9jQJ8nVUzSCnj543z+/AXm+ZM7SYcQWQt3Fymyeg33ouE3Kz4XLS8MJLIOuELwBSKXwOuqm141uNEPGhBLALIME2xjYQdf3NROka5freFYGjshWb1Kz79wk4DlIYxkQLuw8XOQk74f82Q0Nag60dMTnyCwUXE4iWygzW4rhbzbvUOlpVTl3R2wcepO/wEJQElQw2aqscDiJuqEXINLCMRueLOCgbN3Ztvbjdg+Xt+2zMeG5/0a2ADp48rOd/jE9yXoxyai6XnSLl8ujj7AwKhy+YHhdtqRMax34ExIjMY2/HJfXYywZfyie0BliX7+iFMeugCk7+3X3ejQx8TfEI/0ExPeWVYcS+uuTEqVSF84sfvS4+sN7dv2YOoTg59IUPU9esKr+e79xDHHq3F2CcZjIDqPbdjlOoHA5csD60WxkmQ8FT9en4hKVFzbw1JniMgAE=
```

Comment:
why there is several secure items? Need several gpg keys for different parts?

Reply:
those are encrypted env vars needed in the maven settings xml.

Reply:
but why "several"?

Reply:
because there are several:
https://github.com/camel-tooling/camel-language-server/blob/master/cd/mvnsettings.xml

Reply:
what about providing a comment about which variable each of them corresponds (as suggested also in http://www.debonair.io/post/maven-cd/#create-travis-yml:a547b4a31e9ae1ba41fe3873843c9208 )
In case, we need to change just one of them it might be convenient and it will allow to know to what it corresponds